src/components/autocomplete/Autocomplete.ts

```diff
@@ -16,8 +16,10 @@ export default class Autocomplete extends AbstractComponent {
     limit?: IAutocompleteParams["limit"]
   ): Promise<IAutocompleteResult> {
     const params: IAutocompleteParams = { query };
-    if (collection_ids) {
-      params.collection_ids = collection_ids;
+    if (Array.isArray(collection_ids)) {
+      // 'collection_ids' in the autocomplete endpoint expects a comma separated list
+      // as a workaround, we stringify the array before calling the endpoint
+      params.collection_ids = collection_ids.join(',') as any;
```

Comment:
I'd like to avoid the use of `any` wherever possible, even in a pedantic terminal case like this. Keeps our linting rules tight if we'd like to do so. Try this instead:

```suggestion
      params.collection_ids = collection_ids.join(
        ","
      ) as unknown as IAutocompleteParams["collection_ids"];
```